packages/dicomImageLoader/src/imageLoader/wadouri/metaData/metaDataProvider.ts

```diff
@@ -245,6 +246,27 @@ function metaDataProvider(type, imageId) {
       actualFrameDuration: dicomParser.intString(dataSet.string('x00181242')),
     };
   }
+
+  if (type === 'dataset') {
```

Comment:
Doesn't OHIF use 'instance'   Lets try to keep consistent with what is already existing if possible.  I think that is an equivalent type setup, and will allow all tags to be accessed for such providers.